async/retry.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+
+const defaultRetryOptions = {
+  multiplier: 2,
+  maxTimeout: 60000,
+  maxAttempts: 5,
+  minTimeout: 100,
```

Comment:
100 feels a little short to me. What about 1000, which is the default of popular [retry](https://www.npmjs.com/package/retry) module?